src/components/CountryPerformanceOnRisk.js

```diff
@@ -71,13 +93,31 @@ class CountryPerformanceOnRisk extends Component {
 
 
   render() {
+    let graphsToShow = this.state.graphsToShow
```

Comment:
This looks like it is part of #12 (one graph for each risk) - which we planned to do last.